### DISCUSSION TITLE
using nuclei-templates in burp and OWASP ASVS

### DISCUSSION THREAD

**Discussion by moneeraden** [projectdiscovery/nuclei-templates#2247] [Q&A] [answered]
Thanks you guys for your amazing job, i would like to use nuclei-templates through burp, so is there any addon for this, and my other question is there any templates that cover OWASP ASVS levels at least level one
Thanks a lot

**Comment by olearycrew** [comment, marked as answer]
There is now!

https://github.com/projectdiscovery/nuclei-burp-plugin
